<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitForWidget</class>
 <widget class="QWidget" name="WaitForWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title">Edit condition to wait for</string>
  </property>
  <property name="whatsThis">
   <string comment="@info:whatsthis">&lt;para&gt;Edit a condition to wait for until it is met.&lt;/para&gt;
&lt;para&gt;A condition can be a plain condition or a composed condition. Plain conditions wait for some specific thing to happen, for example, they wait until a signal is emitted. Composed conditions contain other conditions (plain or also composed) and wait until something happens in its contained conditions, for example, wait until all the contained conditions were met.&lt;/para&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="waitForWidgetVerticalLayout">
   <item>
    <widget class="AutoExpandableTreeView" name="waitForTreeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KPushButton" name="addButton">
       <property name="toolTip">
        <string comment="@info:tooltip">Add a new condition</string>
       </property>
       <property name="whatsThis">
        <string comment="@info:whatsthis">&lt;para&gt;Adds a new condition.&lt;/para&gt;
&lt;para&gt;A condition has to be added to composed conditions, that is, conditions that contain other conditions, or as the root condition when there is no other condition.&lt;/para&gt;</string>
       </property>
       <property name="text">
        <string comment="@action:button">Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="editButton">
       <property name="toolTip">
        <string comment="@info:tooltip">Edit the selected condition</string>
       </property>
       <property name="whatsThis">
        <string comment="@info:whatsthis">&lt;para&gt;Edits the selected condition.&lt;/para&gt;
&lt;para&gt;Only plain conditions, that is, conditions that are not composed by other conditions, can be edited.&lt;/para&gt;</string>
       </property>
       <property name="text">
        <string comment="@action:button">Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="removeButton">
       <property name="toolTip">
        <string comment="@info:tooltip">Remove the selected condition</string>
       </property>
       <property name="whatsThis">
        <string comment="@info:whatsthis">Removes the selected condition.</string>
       </property>
       <property name="text">
        <string comment="@action:button">Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AutoExpandableTreeView</class>
   <extends>QTreeView</extends>
   <header>AutoExpandableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
